core/src/main/java/org/radargun/RemoteSlaveConnection.java

```diff
@@ -410,12 +410,6 @@ private void startServerSocket() throws IOException {
       }
       serverSocketChannel.socket().bind(address);
       log.info("Master started and listening for connection on: " + address);
-      log.info("Waiting 5 seconds for server socket to open completely");
-      try {
-         Thread.sleep(5000);
```

Comment:
Why are you removing this? Maybe I should ask why it was there before.

Reply:
Yep, the latter is the correct question :) It has been there after some early committer (back at Mircea's days) seemingly resolved some problem. Does not make much sense to me now and I was running happily without.